tests/test_dummy.cpp

```diff
@@ -0,0 +1,4 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#include <venoge/venoge.h>
```

Comment:
Would it make sense to add an actual dummy test to make sure linking and executing are actually ok?

Reply:
Yes, I’m just not sure if it belongs to this PR or a subsequent one (I think I had Catch issues last time I tried to add a non-empty test).

Reply:
I believe it will naturally come in a later PR, I would say.